grammars/css.cson

```diff
@@ -483,6 +483,10 @@
         'include': '#comment-block'
       }
       {
+        'match': '--([A-Za-z0-9_-]+)'
```

Comment:
This needs a more complex pattern in order to avoid matching `------` as a variable, which I'm pretty sure is invalid.  @esdoppio can probably give some tips regarding this.